clients/drcachesim/scheduler/scheduler.h

```diff
@@ -1670,6 +1684,7 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         // Exported statistics. Currently all integers and cast to double on export.
         std::vector<int64_t> stats =
             std::vector<int64_t>(memtrace_stream_t::SCHED_STAT_TYPE_COUNT);
+        uint64_t idle_count = 0;
```

Comment:
I suggest a comment pointing out that this is now a component of simulation time.

Reply:
Added:

        // When no simulation time is passed to us, we use the idle count plus
        // instruction count to measure time.